openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -1677,6 +1677,13 @@ def get_unitless_values(attr):
 
         for attr in identical_attrs:
             this_val, other_val = get_unitless_values(attr)
+
+            # Convert None values to str, so "==" comparison will work
+            if this_val is None:
```

Comment:
does None != None?

Reply:
Good catch. This was originally a quick fix for what I now realize is a validation failure, where we had an OFFXML with `sa_model="None"`, which got eaten as a string. I'll revisit this.

Reply:
Reverted. @andrrizzi 's ParameterAttribute classes handle this now.